README.md

```diff
@@ -186,6 +186,9 @@ git remote add upstream https://github.com/o1-labs/zkapp-cli.git
 npm install
 npm link # makes it available globally on your system
 
+# You can also use the following commands if this is not the fresh clone:
+# npm r zkapp-cli -g && npm install && npm run build --if-present && npm link
```

Comment:
@shimkiv when would you run this command? I wasn't exactly sure from your comment above the command. Could you clarify the comment?

Reply:
So `npm unlink` doesn't work for me at least on `macOS`, hence, I have to do it differently but similarly with the help of `npm r`. Then, when it comes to "when?" - anytime you would like to be sure you linked the correct version after, say, switching the branches.

Reply:
That is what I thought you were trying to say with this comment.  "You can also use the following commands if this is not the fresh clone:".  It would be helpful to clarify it to something like "You can also use the following commands if you have previously run npm link with zkApp cli" or something similar

Reply:
Hope it is better now https://github.com/o1-labs/zkapp-cli/pull/420/commits/3a9342483d23b4e6c0f66c1efe33a6c3fc3af7c1